manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -37,6 +46,15 @@ spec:
         host: istiod.{{ .Values.global.istioNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
         port:
           number: 15012
+  {{- if or .Values.sidecarInjectorWebhook.injectionURL .Values.base.validationURL }}
+  - match:
+    - port: 15053
+    route:
+    - destination:
+        host: istiod.{{ .Release.Namespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
+        port:
+          number: 443
```

Comment:
Can you add a TODO for changing the istiod port to 15053 ( so we don't have to do the 443 redirect ) ? What is the current container port

Reply:
current container port for webhook is 443 maps to target port 15017.

Reply:
I don't understand this todo request tho.

Reply:
Right now we have a Service for Istiod, with service port 443 and container port 15017. 
This is due to istiod no longer running as root - so can't use 443 - and older versions of 
k8s requiring port 443.

If we confirmed that 443 is no longer required by apiserver - we should just use port 15017, without translation from 443.

There is one particular scenario - using nodePort - where we may want to actually use a 
port in the range required for nodePort. In such cases the gateaway is not needed ( I have 
an RFC describing the mode )

Reply:
The port is user facing - users have firewall rules for the port. If we change it yet another time without a good reason, people won't be happy I suspect